### PR TITLE
Added missing plugins to PLUGIN_CATALOG.md and alphabetized

### PR DESCRIPTION
**Pull request by tiffanyfj**
Summary of changes:
- Added missing plugins e.g. Neutron, Cinder, Keystone, KairosDB
- Reordered plugins to be in alphabetical order for easier finding

Testing done:
- Looked at MD output 

@intelsdi-x/snap-maintainers

### CODE REVIEW

**Comment by andrzej-k** [comment]
Great work @tiffanyfj: My only comments are:
- to move disk, elasticsearch, etcd, nfsclient and heka to maintained plugins section
- to add glance to maintained plugins section
- to add cassandra publisher to commited plugins section

File: docs/PLUGIN_CATALOG.md [hunk 1]
```diff
@@ -1,59 +1,64 @@
...
 | CEPH | Collector | Collects from CEPH cluster | [snap-plugin-collector-ceph](https://github.com/intelsdi-x/snap-plugin-collector-ceph) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-ceph)
+| Cinder | Collector | Collects from OpenStack Cinder | [snap-plugin-collector-cinder](https://github.com/intelsdi-x/snap-plugin-collector-cinder) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-cinder)
+| CPU | Collector | Collects CPU metrics from Linux procfs | [snap-plugin-collector-cpu](https://github.com/intelsdi-x/snap-plugin-collector-cpu) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-cpu)
+| DBI | Collector | Collects metrics from SQL DBs using go's "sql" package | [snap-plugin-collector-dbi](https://github.com/intelsdi-x/snap-plugin-collector-dbi) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-dbi)
```
Comment (IzabellaRaulin) [inline docs/PLUGIN_CATALOG.md]: Consider changing the description of snap-plugin-collector-dbi to something like this: "Collects metrics as a results of executing SQL statements on database (MySQL and PostgreSQL supported)"

File: docs/PLUGIN_CATALOG.md [hunk 2]
```diff
@@ -1,59 +1,64 @@
...
+| Facter | Collector | Collects from Facter | [snap-plugin-collector-facter](https://github.com/intelsdi-x/snap-plugin-collector-facter) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-facter) &#124; [Darwin](https://s3-us-west-1.amazonaws.com/snap-plugins-darwin-latest/snap-plugin-collector-facter)
+| HAProxy | Collector | Collects metrics from HAProxy | [snap-plugin-collector-haproxy](https://github.com/intelsdi-x/snap-plugin-collector-haproxy) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-haproxy)
+| InfluxDB | Publisher | Writes to Influx Database | [snap-plugin-collector-influxdb](https://github.com/intelsdi-x/snap-plugin-collector-influxdb) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-influxdb)
```
Comment (IzabellaRaulin) [inline docs/PLUGIN_CATALOG.md]: Here should be about InfluxDB collector which collects internal statistics from Influx database.

File: docs/PLUGIN_CATALOG.md [hunk 3]
```diff
@@ -1,59 +1,64 @@
...
+| Interface | Collector | Collects network interfaces metrics from Linux procfs | [snap-plugin-collector-interface](https://github.com/intelsdi-x/snap-plugin-collector-interface) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-interface)
 | IOstat | Collector | Collect from IOstat | [snap-plugin-collector-iostat](https://github.com/intelsdi-x/snap-plugin-collector-iostat) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-iostat)
+| Keystone | Collector | Collects from OpenStack Keystone | [snap-plugin-collector-ceph](https://github.com/intelsdi-x/snap-plugin-collector-keystone) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-keystone)
```
Comment (IzabellaRaulin) [inline docs/PLUGIN_CATALOG.md]: The text of link is invalid (change snap-plugin-collector-ceph to snap-plugin-collector-keystone)

File: docs/PLUGIN_CATALOG.md [hunk 4]
```diff
@@ -1,59 +1,64 @@
...
 | Libvirt | Collector | Collects from libvirt | [snap-plugin-collector-libvirt](https://github.com/intelsdi-x/snap-plugin-collector-libvirt) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-libvirt)
+| Load | Collector | Collects plaform load metrics from Linux procfs | [snap-plugin-collector-load](https://github.com/intelsdi-x/snap-plugin-collector-load) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-load)
+| Meminfo | Collector | Collects memory related metrics from Linux procfs | [snap-plugin-collector-meminfo](https://github.com/intelsdi-x/snap-plugin-collector-meminfo) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-meminfo)
+| MySQL | Collector | Collects metrics from MySQL DB | [snap-plugin-collector-mysql](https://github.com/intelsdi-x/snap-plugin-collector-mysql) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-publisher-mysql) &#124; [Darwin](https://s3-us-west-1.amazonaws.com/snap-plugins-darwin-latest/snap-plugin-publisher-mysql)
```
Comment (IzabellaRaulin) [inline docs/PLUGIN_CATALOG.md]: Consider adding that the snap-plugin-collector-mysql exposes the **internal statistics**.  Another case is retrieving stored data from database, what can be done only by snap-plugin-collector-dbi

File: docs/PLUGIN_CATALOG.md [hunk 5]
```diff
@@ -1,59 +1,64 @@
...
+| Neutron | Collector | Collect from OpenStack Nova | [snap-plugin-collector-neutron](https://github.com/intelsdi-x/snap-plugin-collector-neutron) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-neutron)
```
Comment (IzabellaRaulin) [inline docs/PLUGIN_CATALOG.md]: In description there is Nova instead Neutron.

**Comment by tiffanyfj** [comment]
@andrzej-k Thanks. I added only what we had binaries for since I had assumed that since we just made the release it included all public plugins. I recall some talk about the Heka plugin, but not the rest.
cc @geauxvirtual 

The Cassandra publisher should be there.

**Comment by andrzej-k** [comment]
How about adding Glance (https://github.com/intelsdi-x/snap-plugin-collector-glance) to the list of maintained plugins? :)
Other then that LGTM.

**Comment by tiffanyfj** [comment]
@andrzej-k Oops, thanks! Fixed.

**Comment by candysmurf** [comment]
LGTM

File: docs/PLUGIN_CATALOG.md [hunk 6]
```diff
@@ -1,60 +1,66 @@
...
-| Etcd | Collector | Collects metrics from Etcd's `/metrics` endpoint. | [snap-plugin-collector-etcd](https://github.com/intelsdi-x/snap-plugin-collector-etcd) | [@danielscottt](https://github.com/danielscottt) |
-| Elasticsearch | Collector | Collects metrics from Elasticsearch cluster | [snap-plugin-collector-elasticsearch](https://github.com/intelsdi-x/snap-plugin-collector-elasticsearch) | [@candysmurf](https://github.com/candysmurf) |
 | Cassandra | Collector | Collects metrics from Cassandra cluster | [snap-plugin-collector-cassandra](https://github.com/intelsdi-x/snap-plugin-collector-cassandra) | [@candysmurf](https://github.com/candysmurf) |
-| HEKA | Publisher | Publishes snap metrics into heka via TCP | [snap-plugin-publisher-heka](https://github.com/intelsdi-x/snap-plugin-publisher-heka) | [@candysmurf](https://github.com/candysmurf) |
+| Glance | Collector | Collects metrics from OpenStack Glance | [snap-plugin-collector-glance](https://github.com/intelsdi-x/snap-plugin-collector-glance) | [@marcin-krolik](https://github.com/marcin-krolik)
```
Comment (IzabellaRaulin) [inline docs/PLUGIN_CATALOG.md]: snap-plugin-collector-glance should be in section `Maintained plugins`.  Others then that LGTM.

**Comment by mjbrender** [comment]
LGTM. Merging it! 🚢

**Pull request merged**